src/backend/database_migrations/versions/20220830_171547_add_pathogen_field.py

```diff
@@ -0,0 +1,92 @@
+"""
+
+Create Date: 2022-08-30 17:15:56.013826
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220830_171547"
+down_revision = "20220816_000535"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    op.add_column(
+        "pathogen_genomes",
```

Comment:
I think our data model had pathogen_id added to samples, I don't think we need it added to pathogen_genomes as well.